### ISSUE TITLE
Couldn't open the pre-trained model of resnet18 from OneDrive

### ISSUE THREAD

**Issue by lampardwk** [bl0/negative-margin.few-shot#11]
Couldn't open the pre-trained model of resnet18 from OneDrive. The error is that the file is corrupted and cannot be opened.
Appreciate your reply, thanks!

**Comment by bl0** [comment]
Hi, I just download the file from OneDrive and it works well. Maybe there is something wrong with your environment. Such as the python version or the pytorch version?

![image](https://user-images.githubusercontent.com/9164121/102843749-d6f86500-4444-11eb-9511-7a548cbffc9e.png)

**Comment by lampardwk** [comment]
Thank you very much for taking time out of your busy schedule to reply！ I got it. But which result in the log file is the final result( in 1 shot and 5shot). Can you share the log file?
Appreciate your reply, thanks!

**Comment by bl0** [comment]
Hi,  I'm sorry that I can't access the original log files since I have checked out.
But you can just run the code and reproduce the results in the paper. If you experience any problems, please feel free to comment here.

**Comment by lampardwk** [comment]
Thank you for your patient reply! I conducted 1-shot and 5-shot experiments on the CUB data set. In the 1-shot experiment, I got the following results:
![1111](https://user-images.githubusercontent.com/42863778/102856888-5ea09c80-4462-11eb-801e-d4e54019e8da.png)
 In the 5-shot experiment, I got the following results:
![2222](https://user-images.githubusercontent.com/42863778/102856923-6e1fe580-4462-11eb-86a0-38e063034c6b.png)

Which data are selected for the final 1shot and 5shot results?

**Comment by bl0** [comment]
Hi, the `1 shot test` results of the best validate model in the 1-shot experiment is selected as the final 1-shot results (In your case, it's 71.43% +- 0.88%). Same with the 5-shot scenario (89.21% +- 0.42%).
Both the two results are slightly worse than the reported in our paper.

**Comment by lampardwk** [comment]
Thanks again for your reply!  In my case:
the 1-shot results in the 5-shot experiment is better than 1-shot results in the 1-shot experiment
the last model is better not the best model for the 5-shot results in the 5-shot experiment.
This seems abnormal, why does this happen?
I'm very sorry, I am a newbie so there are a lot of questions

**Comment by bl0** [comment]
Hi, actually the few-shot accuracy, especially 1-shot accuracy, is very unsteady. And there is some domain difference between validate and test classes, so the best model on the validate classes don't always get the best test few-shot accuracy.

**Comment by lampardwk** [comment]
Oh, I got it!
You are so nice! Thank you for reply!
